static/app/views/profiling/profileSummary/content.tsx

```diff
@@ -101,6 +104,7 @@ function ProfileSummaryContent(props: ProfileSummaryContentProps) {
 
   return (
     <Layout.Main fullWidth>
+      <ProfileCharts query={props.query} router={router} hideCount />
```

Comment:
Why not useRouter inside the charts directly?

Reply:
we can, didn't really cross my mind haha i just blindly fixed the compiler errors 😅

Reply:
fixed this up